sdk/search/azure-search-documents/azure/search/documents/_search_client.py

```diff
@@ -259,11 +270,21 @@ def search(self, search_text, **kwargs):
         search_fields_str = ",".join(search_fields) if search_fields else None
         search_mode = kwargs.pop("search_mode", None)
         query_language = kwargs.pop("query_language", None)
-        speller = kwargs.pop("speller", None)
-        answers = kwargs.pop("answers", None)
+        query_speller = kwargs.pop("query_speller", None)
         select = kwargs.pop("select", None)
         skip = kwargs.pop("skip", None)
         top = kwargs.pop("top", None)
+
+        query_answer = kwargs.pop("query_answer", None)
+        query_answer_count = kwargs.pop("query_answer_count", None)
+        answers = '{}|count-{}'.format(query_answer, query_answer_count)
```

Comment:
We only have this when query_answer_count  is not None, right?

Or we should not contain "|count-" in answers.

Reply:
I pushed a change and verified it works.